.gitignore

```diff
@@ -1,6 +1,8 @@
-dist
+# Allow static/dist to contain empty folders
```

Comment:
this comment is really only relevant for understanding this change, but not for future readers
```suggestion
```